### ISSUE TITLE
Why is there a ".NET Core CLI" help page ?

### ISSUE THREAD

**Issue by StefH** [MicrosoftDocs/vsts-docs#2062]
All information which is defined on this page (https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=vsts) is also explained on (https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core?view=vsts)

So what is the reason for this page ?
Maybe better to remove this page and rename `.NET Core` page to `.NET Core CLI` ?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d227a6f2-a94b-1c05-57e4-b63d42e2b9e5
* Version Independent ID: d063e189-d0fd-4ca3-c108-a0bd2ae49d22
* Content: [.NET Core CLI - Azure Pipelines & TFS](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=vsts)
* Content Source: [docs/pipelines/tasks/build/dotnet-core-cli.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/pipelines/tasks/build/dotnet-core-cli.md)
* Product: **devops**
* GitHub Login: @erickson-doug
* Microsoft Alias: **dastahel**

**Comment by codyconfer** [comment]
Bump this.
I spent hours referring to Azure DevOps > Build Pipelines > Reference > Tasks > Build Tasks > .NET Core CLI trying to solve an issue that had a solution laid out in Azure DevOps > Build Pipelines > Reference > Tasks > Build Tasks > .NET Core. They both reference the same tool. The page separation makes sense if .NET Core Cli stays the same and .NET Core is renamed to .NET Core Cli Usage and the pages are linked with each other internally. 

I created an issue on the task repo while trying to solve the issue and updated when I found the correct documentation.
https://github.com/microsoft/azure-pipelines-tasks/issues/10507
